lldb/docs/resources/test.rst

```diff
@@ -60,7 +60,8 @@ something like ``target.BreakpointCreateByName`` [#]_.
 A good rule of thumb is to prefer shell tests when what is being tested is
 relatively simple. Expressivity is limited compared to the API tests, which
 means that you have to have a well-defined test scenario that you can easily
-match with ``FileCheck``.
+match with ``FileCheck``. Though Shell tests can be run remotely, remote
```

Comment:
I think this would be a little clearer:
"Though Shell tests can be run remotely, behavior specific to remote debugging must be tested with API tests instead."

So that remote specifics are tested only using API tests not in addition to shell tests.

Reply:
Fixed.